rfc/src/mdn.mdown

```diff
@@ -0,0 +1,35 @@
+%%%
+    title = "Sending an MDN Response with JMAP"
```

Comment:
s/an/a

Reply:
nope, see rfc 8098 where it is ofter used